docs/Versioning.md

```diff
@@ -80,6 +82,8 @@ The following are not breaking:
 * Clarifications of specification ambiguities to match prevailing
   implementation practice.
 
+* Adding new type to inputs or outputs.
```

Comment:
How about adding the following? _Earlier such changes were considered breaking changes requiring an increase in the version-number. However, in practice, the cost of increasing the version-number in this situation outweighs the benefits. This is particularly so since backends, in practice, rarely guarantee complete coverage of all types that exist in the standard._